src/main/java/io/github/linoxgh/moretools/MoreTools.java

```diff
@@ -33,18 +31,19 @@ public void onEnable() {
         instance = this;
         config = new Config(this);
         String version = getDescription().getVersion();
-        
+
         if (version.startsWith("DEV")) {
             String cfgVersion = config.getString("version");
             if (cfgVersion != null && !cfgVersion.equals(version)) {
                 saveResource("config.yml", true);
-                
+
                 config = new Config(this);
                 config.setValue("version", version);
                 config.save();
-                
+
                 getServer().getPluginManager().disablePlugin(this);
-                getLogger().log(Level.SEVERE, "Your config.yml file is outdated, resetting it and then disabling the plugin! Please restart.");
+                getLogger().log(Level.SEVERE, "Your config.yml file is outdated, resetting it and then disabling the " +
```

Comment:
Divide this in ```,``` part. So it'll be more readable. Like 1 line ends with ```outdated, ``` and the next starts with `resetting it`.